Classes/common/query/CDTQQuerySqlTranslator.m

```diff
@@ -495,4 +545,32 @@ + (CDTQSqlParts *)selectStatementForAndClause:(NSArray *)clause usingIndex:(NSSt
     return parts;
 }
 
++ (CDTQSqlParts *)selectStatementForTextClause:(NSObject *)clause usingIndex:(NSString *)indexName
+{
+    if (!clause) {
+        return nil;  // no query here
+    }
+    
+    if (!indexName) {
+        return nil;
+    }
+    
+    if (![clause isKindOfClass:[NSDictionary class]]) {
+        return nil; // should never get here as this would not pass normalization
+    }
+    
+    NSDictionary *textClause = (NSDictionary *) clause;
+    NSDictionary *searchClause = textClause[TEXT];
+    
+    NSString *tableName = [CDTQIndexManager tableNameForIndex:indexName];
+    NSString *search = [searchClause[SEARCH] stringByReplacingOccurrencesOfString:@"'"
+                                                                       withString:@"''"];
```

Comment:
And is this required if we're using a placeholder for it? I guess it was for Java?